README.md

```diff
@@ -19,3 +19,94 @@ Frontend Service can be found here -`https://github.com/Emergency-Response-Demo/
   `oc new-app --name find-service --strategy docker https://github.com/Emergency-Response-Demo/find-service`
   
   You will need to pass `INCIDENT_SERVICE_URL` and `MISSION_SERVICE_URL` as Environment variable.
+  
+## Deploying using Tekton Pipeline
+
+* Prerequisite
+
+    OpenShift v4.x Cluster. You can install using `OpenShift Pipelines Operator` from the Operator Hub.
```

Comment:
It would be better to put the pipelines operator requirement in a separate line.